Gopkg.toml

```diff
@@ -6,7 +6,7 @@ required = ["github.com/golang/protobuf/protoc-gen-go"]
 
 [[constraint]]
   name = "github.com/linkerd/linkerd2-proxy-api"
-  revision = "6886fbf5f10ef0cf45978fe8463c7afb717cb467" # not published yet
+  revision = "a9f0b9f267bd027be5648b87389fe46abc38c59e" # not published yet
```

Comment:
Hmm, changing this revision should have modified the `input-digest` hash in Gopkg.lock. Is it possible you manually edited Gopkg.lock? That file is updated automatically as a result of running `bin/dep ensure`. Can you try running `bin/dep ensure` on this branch to see if updates Gopkg.lock?

Reply:
i got into a state where only the input-digest changed and i didnt commit it. this is probably from one of my PRs into this branch...